### PR TITLE
TMEDIA-653-standardize-caching-actions

### PR DESCRIPTION
**Pull request by JackHowa**
- See https://github.com/WPMedia/engine-theme-sdk/pull/602

### CODE REVIEW

**Review by ecarlisle — APPROVED** [review]

**Comment by JackHowa** [comment]
will merge when checks pass

File: package.json [hunk 1]
```diff
@@ -78,6 +77,7 @@
     "jest": "^26.2.0",
     "jest-enzyme": "^7.1.0",
     "lerna": "^4.0.0",
+    "lodash": "^4.17.21",
```
Comment (matthewroach) [inline package.json]: Can we remove this test dependency

**Review by matthewroach — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (JackHowa) [reply package.json]: oh damn yes

Comment (JackHowa) [reply package.json]: will do

File: package.json [hunk 2]
```diff
@@ -59,8 +59,7 @@
     "babel-jest": "^26.2.0",
     "babel-loader": "^8.1.0",
     "babel-plugin-module-resolver": "^3.2.0",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
```
Comment (JackHowa) [inline package.json:63]: adding this back as it's failing our tests

File: package.json (hunk 1 above)
Comment (JackHowa) [reply package.json]: updated

**Review by JackHowa — COMMENTED** [review]
looks good to me

**Pull request merged**